docs/source/tracking.rst

```diff
@@ -382,10 +383,10 @@ Additionally, you should ensure that the ``--backend-store-uri`` (which defaults
 Logging to a Tracking Server
 ----------------------------
 
-To log to a tracking server, set the ``MLFLOW_TRACKING_URI`` environment variable to the server's URI, 
```

Comment:
Are these trailing spaces getting deleted? Was this accidental? More similar changes ...
This would remove space when this documentation is rendered?

Reply:
This was accidental. My editor automatically removes trailing spaces. I can revert the changes. That said, do trailing spaces matter here? A lot of projects do not allow trailing spaces.

Reply:
Removing trailing spaces is fine, if you intend to do that. However, not recommended if ...

1. ... there are other changes you have done in the PR, OR
2. ... removing a space could end up sticking to words or a word and a punctuation and a word on the next line (like in this case, "URI", a comma, and the word "along" on the next line). When this documentation is rendered it will look weird.